lib/galaxy/datatypes/converters/vcf_bgzip_to_vcf_converter.xml

```diff
@@ -0,0 +1,12 @@
+<tool id="CONVERTER_vcf_bgzip_to_vcf_0" name="Convert VCF_BGZIP to VCF" version="1.0.0" hidden="true">
+    <!-- <description>__NOT_USED_CURRENTLY_FOR_CONVERTERS__</description> -->
```

Comment:
We should also add explicit requirements so that the converter can run containerized.
I'd use
```
<requirements>
    <requirement type="package" version="0.15.2">pysam</requirement>
    <requirement type="package" version="3.7">python</requirement>
</requirements>
```